### PR TITLE
[CLASS UPDATE] Added Path Distance Calculations.

### PR DESCRIPTION
**Pull request by Fryslan**

### CODE REVIEW

File: src/main/java/org/rev317/min/api/interfaces/TileFlags.java [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 public interface TileFlags {
 
-    public int WALL_NORTHWEST = 0x1;
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/interfaces/TileFlags.java]: These should be public

File: src/main/java/org/rev317/min/api/methods/Calculations.java [hunk 2]
```diff
@@ -23,7 +22,6 @@ public static final double distanceTo(Tile tile) {
      *
      * @param a
      * @param b
-     *
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/methods/Calculations.java]: Don't remove the whitelines, they should be there, as it provides readability

**Comment by JKetelaar** [comment]
Added a few comments, please fix those first.

**Comment by CapslockRSPS** [comment]
This should a method in the API.

`/**
     * @author El Maestro
     */
    public static boolean areTilesTheSame(Tile first, Tile second) {
        return first.getX() == second.getX() && first.getPlane() == second.getPlane() && first.getY() == second.getY();
    }`

**Comment by JKetelaar** [comment]
Could you form that code into proper aligned code, looks a mess now.

Make sure there are whitelines before/after the ```

**Comment by JKetelaar** [comment]
Great method, should have a different name though...

**Comment by CapslockRSPS** [comment]
Not my naming. Like the comment says the author was El Maestro.

**Comment by Fryslan** [comment]
Will do, isnt this better coding : return first.equals (second) ;  or wont that work?

**Comment by JKetelaar** [comment]
Nope, that's not the reason of the method

**Comment by Fryslan** [comment]
Then what is th reason?
All i see is checking if two tiles have the same data.

**Comment by JKetelaar** [comment]
Object comparison isn't that precisely

**Comment by Fryslan** [comment]
A oke, will add the method.

File: src/main/java/org/rev317/min/api/methods/Equipment.java [hunk 3]
```diff
@@ -0,0 +1,134 @@
+package org.rev317.min.api.methods;
+
+import org.parabot.environment.api.utils.Time;
+import org.parabot.environment.scripts.framework.SleepCondition;
+import org.rev317.min.Loader;
+import org.rev317.min.api.wrappers.Item;
+
+/**
+ * Created by Piet Jetse Heeringa on 27-6-2016.
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/methods/Equipment.java]: Use `@author Piet Jetse Heeringa` instead

File: src/main/java/org/rev317/min/api/methods/Equipment.java [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+                @Override
+                public boolean isValid() {
+                    return isWearing(id);
+                }
+            }, 3000);
+        }
+    }
+
+    /**
+     * UnEquips an Item based on the ID of that item.
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/methods/Equipment.java]: UnEquips should be Unequips

File: src/main/java/org/rev317/min/api/methods/Equipment.java [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    public static void unequip(int id) {
+        for (Slot slot : Slot.values()) {
+            if (slot.id() == id) {
+                unequip(slot);
+            }
+        }
+    }
+
+    /**
+     * UnEquips the item from the given Slot.
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/methods/Equipment.java]: UnEquips should be Unequips

File: src/main/java/org/rev317/min/api/methods/Equipment.java [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+     *
+     * @param slot Slot to UnEquip the item from.
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/methods/Equipment.java]: UnEquips should be Unequips

File: src/main/java/org/rev317/min/api/methods/Equipment.java [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+                    return isWearing(id);
+                }
+            }, 3000);
+        }
+    }
+
+    /**
+     * UnEquips an Item based on the ID of that item.
+     *
+     * @param id Item ID to UnEquip.
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/api/methods/Equipment.java]: UnEquips should be Unequips

**Comment by JKetelaar** [comment]
Could you check my latest comments and don't forget [these](https://github.com/Parabot/Parabot-317-API-Minified/pull/47/files#diff-7a2764867895c4b46162d1f5d5265c1cL26) either. Thanks!

**Pull request merged**